icon/README.md

```diff
@@ -15,4 +15,13 @@ This space is administered by:
 Bologna, italy
 <bruno.sartini3@unibo.it>  
 GitHub: [br0ast](https://github.com/br0ast)
-ORCID: [0000-0002-9152-4402](https://orcid.org/0000-0002-9152-4402)  
\ No newline at end of file
+ORCID: [0000-0002-9152-4402](https://orcid.org/0000-0002-9152-4402)  
+
+and
+
+**Sofia Baroncini** <br>
```

Comment:
Note that you can use `  ` (two space characters) at the end of a Markdown line to the get same effect as `<br>`.

Also note that while the new info about `Sofia Baroncini` includes `<br>` at the end of each intended line, the previously existing info about `Bruno Sartini` does not include `<br>` at any line end, and two lines end with *one* space, rather than two, so [when the Markdown is rendered to a web browser](https://github.com/perma-id/w3id.org/blob/cb261354aa6b250cdc0d8934cd800fffee15e01d/icon/README.md), there will be no linewrap between `italy` and `<bruno`, nor between `br0ast)` and `ORCID:`.